### PR TITLE
chore(release): 5.0.1

### PR DESCRIPTION
**Pull request by niklasad1**
Prepare to release 5.0.1

- [x] update deps
    - bump @polkadot/api 4.8.1
    - bump @http/errors 1.8.0
    - bump @polkadot/apps-config 0.90.1
- [x] run tests
- [x] changelog

### CODE REVIEW

**Comment by niklasad1** [comment]
I think we should automate Tarik's test running by including it by a submodule or something.

Then to provide a script to run it.

Perhaps make it possible to "label release PRs" and run it only on those instead of trusting individuals to run it manually ^^

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,25 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
+## [5.0.1](https://github.com/paritytech/substrate-api-sidecar/compare/v5.0.0...v5.0.1) (2021-05-03)
+
+
+### Bug Fixes
+
+* docs: update release and publishing
```
Comment (TarikGul) [inline CHANGELOG.md:11]:
```suggestion
* docs: update release and publishing instructions ([#529](https://github.com/paritytech/substrate-api-sidecar/pull/529))([da33c7f](https://github.com/paritytech/substrate-api-sidecar/commit/da33c7f3a5ab1c0f1c96129736ffbe3285d25573))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,25 @@
...
+
+* fix: readme docs for node version 14
+
+* fix: remove dependabot and add upgrade-interactive
```
Comment (TarikGul) [inline CHANGELOG.md:15]:
```suggestion
* fix: remove dependabot and add upgrade-interactive plugin ([#531](https://github.com/paritytech/substrate-api-sidecar/pull/531)) ([f22cf3f](https://github.com/paritytech/substrate-api-sidecar/commit/f22cf3ff6ae292144018821fbaa0c21d6f96490d))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,25 @@
...
 
+
+## [5.0.1](https://github.com/paritytech/substrate-api-sidecar/compare/v5.0.0...v5.0.1) (2021-05-03)
+
+
+### Bug Fixes
+
+* docs: update release and publishing
+
+* fix: readme docs for node version 14
```
Comment (TarikGul) [inline CHANGELOG.md:13]:
```suggestion
* fix: readme docs for node version 14 ([#526](https://github.com/paritytech/substrate-api-sidecar/pull/526)) ([409b611](https://github.com/paritytech/substrate-api-sidecar/commit/409b611d81b6d65c2424b00e4e290d6241ee96f6))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,25 @@
...
+
+* fix: remove dependabot and add upgrade-interactive
+
+* deps: update @polkadot/types 4.8.1
```
Comment (TarikGul) [inline CHANGELOG.md:17]:
```suggestion
* deps: update @polkadot/api for the latest parachain support
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,25 @@
...
+
+* deps: update @polkadot/apps-config 0.90.1
```
Comment (TarikGul) [inline CHANGELOG.md:19]:
```suggestion
* deps: Update @polkadot/apps-config to get latest chain specific upgrades.
```

**Review by TarikGul — APPROVED** [review]
LGTM just some additions to the changelog.

I agree we should automate my test library. Honestly not sure the best way. Assuming some type of docker image that runs a script to run sidecar and the test library side by side. But any suggestions are welcome as I am not perfectly sure.

File: CHANGELOG.md (hunk 4 above)
Comment (niklasad1) [reply CHANGELOG.md:17]: ah, nice catch

**Review by emostov — APPROVED** [review]

**Pull request merged**